addons/website_slides/models/slide_channel.py

```diff
@@ -783,23 +788,29 @@ def _remove_membership(self, partner_ids):
         if not partner_ids:
             raise ValueError("Do not use this method with an empty partner_id recordset")
 
-        earned_karma = self._get_earned_karma(partner_ids)
-        users = self.env['res.users'].sudo().search([
-            ('partner_id', 'in', list(earned_karma)),
-        ])
-        for user in users:
-            if earned_karma[user.partner_id.id]:
-                user.add_karma(-1 * earned_karma[user.partner_id.id])
-
         removed_channel_partner_domain = []
         for channel in self:
+            earned_karma = channel._get_earned_karma(partner_ids)
```

Comment:
Now that I think about it it would probably be much easier to compute the earned karma based on the trackings.

But as we are probably going to remove that part when we archive the membership instead of unlinking, let us keep it like that.
(And also trackings did not exist before so we would need a fallback, annoying).